### PR TITLE
style: simplify string formatting for readability

### PR DESCRIPTION
**Pull request by hamirmahal**
# Objective

The goal of this change is to improve code readability and maintainability.

### CODE REVIEW

File: examples/3d/irradiance_volumes.rs [hunk 1]
```diff
@@ -351,12 +351,7 @@ impl AppStatus {
 
         Text::from_section(
             format!(
-                "{}\n{}\n{}\n{}\n{}",
-                CLICK_TO_MOVE_HELP_TEXT,
-                voxels_help_text,
-                irradiance_volume_help_text,
-                rotation_help_text,
-                switch_mesh_help_text
+                "{CLICK_TO_MOVE_HELP_TEXT}\n{voxels_help_text}\n{irradiance_volume_help_text}\n{rotation_help_text}\n{switch_mesh_help_text}"
```
Comment (alice-i-cecile) [inline examples/3d/irradiance_volumes.rs:354]: This is quite hard to read. Can we put new line literals in strings in Rust?

**Review by alice-i-cecile — APPROVED** [review]
I like all of these other than the irradiance_volumes change, but I don't mind it enough to force inconsistency.

File: examples/3d/irradiance_volumes.rs (hunk 1 above)
Comment (hamirmahal) [reply examples/3d/irradiance_volumes.rs]: You're right, that was harder to read.

I fixed merge conflicts and [redid this part](https://github.com/bevyengine/bevy/pull/15033/files#diff-742f4b1cc2d88a5e36b1880053dcaa2076ffe154d9588e3e41e65e1b6cf4a75eR354-R358).

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Perfect, thanks!

**Comment by hamirmahal** [comment]
You're welcome!

**Pull request merged**